config.sh

```diff
@@ -135,4 +135,6 @@ elif [ `uname -s` != "OpenBSD" ]; then
 		check_custom "ncursesw6" "ncursesw6-config" || fail "ncursesw"
 fi
 check_ssl_implementation
+echo "# Configuration for catch" >> config.mk
+echo "CXXFLAGS+=-DCATCH_AMALGAMATED_CUSTOM_MAIN" >> config.mk
```

Comment:
`config.mk` contains machine-specific flags such as paths to some libraries, so it's not a good place for this define. I'd put it in `DEFINES` in the Makefile.

Reply:
Fixed.